NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -273,6 +278,13 @@ void NeuralAmpModeler::ProcessBlock(iplug::sample** inputs, iplug::sample** outp
   const size_t numFrames = (size_t)nFrames;
   const double sampleRate = GetSampleRate();
 
+  if (const bool namActive = GetParam(kNamToggle)->Value(); !namActive)
+  {
+    _ProcessOutput(inputs, outputs, numFrames, numChannelsInternal, numChannelsExternalOut);
+    _UpdateMeters(inputs, outputs, numFrames, numChannelsInternal, numChannelsExternalOut);
+    return;
```

Comment:
[Nit] I don't really like early returns like this as a matter of style.

However, the rest of this function is pretty sprawling and I don't like that either, so indenting it all in a giant pair of braces also isn't great.

Keep it like it is for this PR, but...justa  note to self to clean this up I guess :)

Reply:
Yes, I don't like it either.
But as you mentioned, I wanted to avoid huge blocks of code wrapped in new if statements and another level of indentation.